Search FAQ update (#4542)
DataDog/documentation: content/en/integrations/faq/container-integration-event.md

---
title: Container Integration Events
kind: faq
---

When monitoring containers and orchestrators (Docker, ECS, and Kubernetes) you are familiar with the events that are available once an integration has been enabled and respective configuration steps followed.

As your containerized environment grows, containers and orchestrators start emitting a high volume of events that leads to more noise in the [events stream][1], [monitors][2], and overlays.

> To ensure that you don't have to worry about the eventual noise, events provided by default are limited to only severe and important events listed below.

Reviewer: ```suggestion
In order to reduce noise, only "severe" and "important"-level events are listed by default.
```
